src/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface ThemeContextInterface {
+  mode?: Mode;
+  toggleMode?: any;
```

Comment:
Let's not use the type `any` anywhere and have always the exact type we need, for example here we need `() => void`